### ISSUE TITLE
Impossible to create graph with query containing asterisk (field wildcard)

### ISSUE THREAD

**Issue by t0mk** [influxdata/chronograf#2369]
When adding graph to dashboard, when I try to store successfully displayed query
SELECT mean("*") FROM "mytr"."autogen".":meas:" WHERE time > :dashboardTime: GROUP BY :interval: FILL(null)

The Chronograf javascript removes the asterisk from "mean("*")"m the query is stored without it, and then I get
```
Error message received status code 400 from server: err: error parsing query: invalid number of arguments for mean, expected 1, got 0" component=server http_status =400
```

**Comment by goller** [comment]
Hi @t0mk thanks for writing in!  I suspect that the javascript isn't handling the wildcard type field.  The chronograf server analyzes the query, substitutes the template variables and returns a structured JSON representation.  The representation of wildcard is probably not understood in the javascript.

I'll take a look.

**Comment by goller** [comment]
Hey @t0mk I have a fix in #2386. It should ship in 1.3.11.

**Comment by t0mk** [comment]
great, thanks! I will test it as soon as it on the chornograf official
docker image
